src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -67,6 +68,11 @@ public ReadOnlyAddressBook getAddressBook() {
         return model.getAddressBook();
     }
 
+    @Override
+    public ObservableList<Event> getEventList() {
+        return model.getCalendar().getEventList();
```

Comment:
Might want to consider Law of Demeter principle.

Reply:
Good idea, has been resolved.